docs/src/pages/premium-themes/tweeper/components/atoms/AppBar.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import cx from 'classnames';
+import MuiAppBar from '@material-ui/core/AppBar';
+import { APP_BAR } from '../../theme/core';
+
+const AppBar = ({ className, shaded, ...props }) => (
+  <MuiAppBar
+    src={'http://i.pravatar.cc/300'}
```

Comment:
This is causing insecure mixed content errors on Netlify...

Reply:
I fixed it by using my avatar url from twitter, is that ok?

https://pbs.twimg.com/profile_images/1060539954361622533/-9ofKMvA_400x400.jpg